docs/05-environments/03-testnet/index.md

```diff
@@ -48,6 +49,7 @@ The faucet API allows users to programmatically interact with the faucet.
 | ERC20          | ERC20TST      | FeeHandlerWithOracle | Mint/Burn      | 0x0000000000000000000000000000000000000000000000000000000000000000 | 0x3D151A97A446C9ea6893038e7C0db73466f3f3af | 0x2465c8F84bDB7130ACDf31d694bc9c820F70ac06 | 0xAc693E44E1EDe5f66A4e1406F65b904450932fB3 |
 | ERC721         | ERC721TST     | BasicFeeHandler      | Mint/Burn      | 0x0000000000000000000000000000000000000000000000000000000000000200 | 0xe9d3b1433bACDfC26ee097629D238A41BF6dA3aE | 0x4beD477d1f5D338855A521ABa2A88c9a15e2eA5d | 0x3D151A97A446C9ea6893038e7C0db73466f3f3af |
 | GenericMessage | N/A           | BasicFeeHandler      | Message        | 0x0000000000000000000000000000000000000000000000000000000000000500 | 0x530Ca8291856c727cc6a33c2ACD50f79184AFA3d | 0xe255cA458925c26d3E05004e247579A64b020cEF | 0xe7Ed7AAd072ACd23bA36F906C2515DF8eD43d482 |
+| GenericMessage | N/A           | BasicFeeHandler      | Message        | 0x0000000000000000000000000000000000000000000000000000000000000500 | 0x530Ca8291856c727cc6a33c2ACD50f79184AFA3d | 0xe255cA458925c26d3E05004e247579A64b020cEF | 0xe7Ed7AAd072ACd23bA36F906C2515DF8eD43d482 |
```

Comment:
@waymobetta Isn't this just a duplicate of already existing GenericMessage - one row up?

Reply:
Thanks, yep, oversight; I was using this as a template to follow for the additional tokens found within the Notion page that we elected not to include.